src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -70,6 +71,7 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
     'developer-token': getValueFromMessage(metadata, 'secret.developer_token'),
   };
   response.params = { event, customerId: filteredCustomerId };
+
   if (subAccount) {
```

Comment:
here add additional condition to execute this logic only if `!isDefined(configDetails.loginCustomerId)`

Reply:
added